master/buildbot/changes/gerritchangesource.py

```diff
@@ -313,7 +314,8 @@ def startStreamProcess(self):
             args = args + ['-i', self.identity_file]
         self.process = reactor.spawnProcess(
             self.LocalPP(self), "ssh",
-            ["ssh"] + args + ["gerrit", "stream-events"])
+            ["ssh"] + args + ["gerrit", "stream-events"],
+            env={'PATH': os.environ.get('PATH', '')})
```

Comment:
looks like this is a weirdness of twisted. I was sure spawnProcess would inherit parent process environment.
but its not, according to doc, unless you *set* env=None
http://twistedmatrix.com/documents/current/api/twisted.internet.interfaces.IReactorProcess.html
I think it would be more consistent with behaviour of the worker if we rather set env=None

There might be more environement variable than PATH that ssh might want to use